scripts/generate-sbom-data.ts

```diff
@@ -46,19 +46,30 @@ const fetchSbomData = async (repo: `${string}/${string}`, retries = 3) => {
 };
 
 async function main() {
-  await fs.mkdir(sbomFolderPath, { recursive: true });
+  const env = process.env.NODE_ENV;
 
-  const feSbom = await fetchSbomData("ohcnetwork/care_fe");
-  await fs.writeFile(
-    path.join(sbomFolderPath, "care_fe-sbom.json"),
-    JSON.stringify(feSbom),
-  );
+  if (!env || !["production", "staging", "develop"].includes(env)) {
+    console.log(`Skipping SBOM fetching in ${env} environment.`);
+    return;
+  }
```

Comment:
```suggestion
  if (!env || !["production", "staging", "development"].includes(env)) {
    console.log(`Skipping SBOM fetching in ${env} environment.`);
    return;
  }
```

if i am not wrong! it should be `development`

Reply:
No. env represents the Git branch, so it should be develop.

Reply:
are you sure?

Reply:
Yep :)

![image](https://github.com/user-attachments/assets/b3e34a2d-8e7f-4715-b80c-f82cbeaff9d2)

Reply:
from what i understand, `env` is `NODE_ENV` and `NODE_ENV` is not your git branch. Are you 100% sure `NODE_ENV` is what you are looking for?

Reply:
Good point, it’s making me second-guess too 🤔. Here’s what I tested:

- Preview mode (https://deploy-preview-10595.preview.ohc.network/)

![image](https://github.com/user-attachments/assets/f0313107-c667-4fa2-9b35-d983a4db67f6)

- Develop branch (locally)

![image](https://github.com/user-attachments/assets/1143c6aa-8bec-49af-9fdd-a38ab42d5d15)

- Terminal check

![image](https://github.com/user-attachments/assets/35618a13-00a8-4f40-a8ab-aa1c31d33bac)


Should probably get @tellmeY18 approval too